### ISSUE TITLE
`LineLengthFixer` doesn't shorten constructor with property promotion

### ISSUE THREAD

**Issue by simoheinonen** [symplify/coding-standard#74]
With
```php
    ->withConfiguredRule(
        LineLengthFixer::class,
        [
            'line_length' => 120,
            'break_long_lines' => true,
            'inline_short_lines' => true,
        ]
    )
```

## Doesn't work
```php
<?php

class Foo
{
    public function __construct(
        private stdClass $foo,
    ) {}
}
```
>  [OK] No errors found. Great job - your code is shiny in style!

## Works
```php
<?php

class Foo
{
    public function __construct(
        stdClass $foo,
    ) {}
}
```
```diff
-    public function __construct(
-        stdClass $foo,
-    ) {}
+    public function __construct(stdClass $foo,) {}
```

**Comment by TomasVotruba** [comment]
I'm looking into this... 

We already have `StandaloneLinePromotedPropertyFixer` that puts promoted property on newline. Using both `StandaloneLinePromotedPropertyFixer` and `LineLenghtFixer` would create infinity loop, that's why its skipped in `LineLenghtFixer`.

I suggest using `StandaloneLinePromotedPropertyFixer` as property promotion is more visible and hardere to hide. Also git diffs are easier in case of addingd new properties. Or make a custom rule to fit your needs.
